ArduPlane/wscript

```diff
@@ -34,7 +34,7 @@ def build(bld):
             'AP_Landing',
             'AP_Beacon',
             'PID',
-            'AP_Soaring'
+            'AP_Soaring',
```

Comment:
> however in all other places you put a comma on last line

Yes, that's not a problem, but we should avoid making these changes when the code isn't being changed for anything else - makes it harder for rebases and cherry-picks.